### DISCUSSION TITLE
standalone lua (readbean) redbean interpreter

### DISCUSSION THREAD

**Discussion by bentxt** [jart/cosmopolitan#1101] [General]
Hi
Readbean extends lua with many very useful functions. Is it somehow possible to use these in a simple script without running it in a redbean/server environment? 

something like `redscript ./hello.lua`

Or are those functions somewhere defined in common lua modules?

Thanks for all the great work!
Ben

**Comment by solisoft** [comment]
You can run ./redbean -i ./hello.lua

**Comment by wildheart** [comment]
There is a standalone lua program at cosmo.zip.
